assets/xml/objects/object_zl4.xml

```diff
@@ -1,125 +1,177 @@
+        <Texture Name="gDmZl4HairTex" OutName="hair" Format="rgba16" Width="16" Height="8" Offset="0x29F0" />
+
+        <!-- Eye Textures -->
+        <Texture Name="gDmZl4EyeOpenLookingUpRightTex" OutName="eye_open_looking_up_right" Format="ci8" Width="32" Height="32" Offset="0x2AF0" TlutOffset="0x870"/>
+        <Texture Name="gDmZl4EyeOpenHalf2Tex" OutName="eye_half_2" Format="ci8" Width="32" Height="32" Offset="0x2EF0" TlutOffset="0x870"/>
+        <Texture Name="gDmZl4EyeOpenClosedTex" OutName="eye_closed" Format="ci8" Width="32" Height="32" Offset="0x32F0" TlutOffset="0x870"/>
+        <Texture Name="gDmZl4EyeWideTex" OutName="eye_wide" Format="ci8" Width="32" Height="32" Offset="0x36F0" TlutOffset="0x870"/>
+        <Texture Name="gDmZl4EyeHalf1Tex" OutName="eye_half_1" Format="ci8" Width="32" Height="32" Offset="0x3AF0" TlutOffset="0x870"/>
+        <Texture Name="gDmZl4EyeOpenLookingLeftTex" OutName="eye_open_looking_left" Format="ci8" Width="32" Height="32" Offset="0x3EF0" TlutOffset="0x870"/>
+        <Texture Name="gDmZl4EyeOpenLookingRightTex" OutName="eye_open_looking_right" Format="ci8" Width="32" Height="32" Offset="0x42F0" TlutOffset="0x870"/>
+
+        <!-- Mouth Textures -->
+        <Texture Name="gZl4MouthNeutralTex" OutName="mouth_neutral" Format="ci8" Width="32" Height="32" Offset="0x46F0" TlutOffset="0xA70"/>
+        <Texture Name="gZl4MouthOpenSmilingTex" OutName="mouth_open_smiling" Format="ci8" Width="32" Height="32" Offset="0x4AF0" TlutOffset="0xA70"/>
+        <Texture Name="gZl4MouthFrowningTex" OutName="mouth_frowning" Format="ci8" Width="32" Height="32" Offset="0x4EF0" TlutOffset="0xA70"/>
+        <Texture Name="gZl4MouthOpenTex" OutName="mouth_open" Format="ci8" Width="32" Height="32" Offset="0x52F0" TlutOffset="0xA70"/>
+
+        <Texture Name="gDmZl4FingersTex" OutName="fingers" Format="rgba16" Width="16" Height="16" Offset="0x56F0" />
+        <Texture Name="gDmZl4BraceletTex" OutName="bracelet" Format="ci8" Width="16" Height="16" Offset="0x58F0" />
+        <Texture Name="gDmZl4BlueUndershirtGradient1Tex" OutName="blue_undershirt_gradient" Format="rgba16" Width="8" Height="8" Offset="0x59F0" />
+        <Texture Name="gDmZl4BlueUndershirtGradient2Tex" OutName="blue_undershirt_gradient_2" Format="rgba16" Width="16" Height="16" Offset="0x5A70" />
+        <Texture Name="gDmZl4StripedShoulderTex" OutName="striped_shoulder_patern" Format="ci8" Width="8" Height="8" Offset="0x5C70" />
+        <Texture Name="gDmZl4UndershirtWrinklesTex" OutName="undershirt_wrinkles" Format="ci8" Width="16" Height="16" Offset="0x5CB0" />
+        <Texture Name="gDmZl4PendantTex" OutName="pendant" Format="rgba16" Width="32" Height="32" Offset="0x5DB0" />
+
+        <DList Name="gDmZl4ChestDl" Offset="0xA630" />
+        <DList Name="gDmZl4HeadDl" Offset="0xAFC8" />
+        <DList Name="gDmZl4RightUpperArmDl" Offset="0xB8D8" />
+        <DList Name="gDmZl4RightForArmDl" Offset="0xBB40" />
```

Comment:
Did you mean "forearm" here?

Reply:
ah yeah, sometimes writing in camel case for a long time my hands cramp up, will fix